src/web/app/components/simple-modal/simple-modal.component.html

```diff
@@ -30,7 +30,7 @@ <h5 class="modal-title">
   <button type="button" class="btn btn-light modal-btn-cancel" *ngIf="!isInformationOnly" (click)="activeModal.dismiss()">
     {{ cancelMessage }}
   </button>
-  <button type="button" class="btn modal-btn-ok"
+  <button type="button" class="btn modal-btn-ok z-index-1"
```

Comment:
Instead of class ```z-index-1```, how about ```style="z-index: 1"``` just to be cleaner

Reply:
I was also considering that! 

However, I saw on line 13 (in the same file):  
```html
<span class="margin-left-5px" [innerHTML]="header"></span>
```
So I thought it would be better to be consistent with the style used in the original code.